docs/examples/arrow_example.ipynb

```diff
@@ -195,42 +269,50 @@
    "source": [
     "### Convert the Arrow data to power-grid-model input data\n",
     "\n",
-    "No direct conversion from Arrow Tables to NumPy exists and a copy is always required.\n",
+    "The Arrow record batch or tables can then be converted to row based data or columnar data.\n",
+    "Converting Arrow data to columnar NumPy arrays is recommended to leverage the columnar nature of Arrow data. \n",
+    "This conversion can be done with zero-copy operations.\n",
+    "\n",
+    "A similar approach be adopted by the user to convert to row based data.\n",
     "\n",
-    "To ensure support for optional arguments and to prevent version lock, it is recommended to create an empty power-grid-model data set using `initialize_array` and then fill it with the Arrow data."
+    "```{note}\n",
+    "The option of `zero_copy_only` in the function below and assert for correct dtype is added in this demo to verify no copies are made. \n",
+    "Its usage is not mandatory to do zero copy conversion.\n",
+    "```"
```

Comment:
this note is outdated since you removed the `zero_copy_only`. either update and/or add a comment in the function below in the code

Reply:
```suggestion
    "```{note}\n",
    "The option of `zero_copy_only` in the `to_numpy` function below and assert for correct dtype is added in this demo to verify no copies are made. \n",
    "Its usage is not mandatory to do zero copy conversion.\n",
    "```"
```
I had it added also because unlike numpy's `copy=False`, pyarrow's `zero_copy_only=True` is a hard check which raises error. So an extra note that such hard check is not mandatory.

Reply:
just an inline comment will do then